src/components/views/CreateGame.js

```diff
@@ -1,6 +1,8 @@
 import React, {useState} from 'react';
 import {useHistory} from 'react-router-dom';
 import BaseContainer from "components/ui/BaseContainer";
+import {api, handleError} from 'helpers/api';
+import Lobby from "models/Lobby";
```

Comment:
I think should be: 

import Lobby from "./Lobby";

I get an error with "models/Lobby"